LibreHardwareMonitorLib/Hardware/Gpu/NvidiaGpu.cs

```diff
@@ -398,6 +398,7 @@ public NvidiaGpu(int adapterIndex, NvApi.NvPhysicalGpuHandle handle, NvApi.NvDis
 
                                         _gpuDedicatedMemoryUsage = new Sensor("D3D Dedicated Memory Used", memorySensorIndex++, SensorType.SmallData, this, settings);
                                         _gpuSharedMemoryUsage = new Sensor("D3D Shared Memory Used", memorySensorIndex, SensorType.SmallData, this, settings);
+                                        _memoryLoad = new Sensor("GPU Memory", nodeSensorIndex++, SensorType.Load, this, settings);
```

Comment:
This should be done differently, if there's no D3DDisplayDevice then this sensor will never be created. It would be better to make sure the sensors here start at another offset (one more).

Reply:
How can you have a GPU with dedicated memory (which is what we are measuring here) and not have a D3DDisplayDevice?

Reply:
D3DDisplayDevice isn't always able to find a matching device, and it's not available on Linux either.